dpctl/tensor/libtensor/source/simplify_iteration_space.hpp

```diff
@@ -87,6 +87,39 @@ void simplify_iteration_space_3(int &,
                                 py::ssize_t &,
                                 py::ssize_t &);
 
+void simplify_iteration_space_4(int &,
```

Comment:
One day, it would be nice to figure out how to make this nicer, perhaps by packing data about `src1`, etc. into structs.

Reply:
And of course, this same comment applies to `simplify_iteration_space_3` and others.